horizomer/utils/tree.py

```diff
@@ -62,9 +47,43 @@ def order_nodes(tree, increase=True):
     return res
 
 
-if __name__ == "__main__":
-    main()
+# replace siblings using children would cause ordering problems
+# assign prev to None for non_tips() would ignore root
+def is_nodes_ordered(tree, increase = True):
+    """Returns 'True' if the tree is ordered.
+
+    Parameters
+    ----------
+    tree : skbio.TreeNode
+        tree to check ordering
+    increase : bool, optional
+        check if nodes in increasing (True) or decreasing (False) order
+
+    Returns
+    -------
+    bool
+        'True' if the tree is ordered
```

Comment:
I would write `whether the tree is ordered` so that we don't have to define `True` and `False` separately.